test-jest/polar/RadialBar.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import { Surface, RadialBar } from '../../src';
+
+describe('<RadialBar />', () => {
+  const data = [
+    { cx: 250, cy: 250, innerRadius: 0, outerRadius: 0, startAngle: 0, endAngle: 60, value: 50 },
+    { cx: 250, cy: 250, innerRadius: 30, outerRadius: 40, startAngle: 0, endAngle: 60, value: 50 },
+    { cx: 250, cy: 250, innerRadius: 50, outerRadius: 60, startAngle: 0, endAngle: 60, value: 50 },
+    { cx: 250, cy: 250, innerRadius: 70, outerRadius: 80, startAngle: 0, endAngle: 60, value: 50 },
+  ];
+
+  test('Render 5 sectors', () => {
+    const { container } = render(
+      <Surface width={500} height={500}>
+        <RadialBar isAnimationActive={false} cx={250} cy={250} data={data} label dataKey="value" />
+      </Surface>,
+    );
+
+    expect(container.querySelectorAll('.recharts-radial-bar-sector')).toHaveLength(data.length);
+  });
+
+  test("Don't render any sector when data is empty", () => {
+    const { container } = render(
+      <Surface width={500} height={500}>
+        <RadialBar data={[]} dataKey="value" />
+      </Surface>,
+    );
+
+    expect(container.querySelectorAll('.recharts-pie')).toHaveLength(0);
+  });
+
+  test('Render 5 sectors', () => {
```

Comment:
It looks like this test name isn't accurate on what its testing. Something like this could be more suitable
```suggestion
  test('RadialBar event handlers', () => {
```

Reply:
agreed though it was named this before, a non-blocking nit

Reply:
but actually I will call it blocking because we shouldn't have more than 1 test with the same name (and the above difference in test)